<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="95728" started-at="2019-11-13T09:33:24Z" finished-at="2019-11-13T09:34:59Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="95728" started-at="2019-11-13T09:33:24Z" finished-at="2019-11-13T09:34:59Z">
      <class name="rctiplus.mainClass">
        <test-method status="PASS" signature="setUp()[pri:0, instance:rctiplus.mainClass@548e7350]" name="setUp" is-config="true" duration-ms="11201" started-at="2019-11-13T16:33:24Z" finished-at="2019-11-13T16:33:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="CekShare_line()[pri:14, instance:rctiplus.mainClass@548e7350]" name="CekShare_line" duration-ms="83084" started-at="2019-11-13T16:33:35Z" finished-at="2019-11-13T16:34:58Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at rctiplus.share_line.cekshare_line(share_line.java:50)
	at rctiplus.mainClass.CekShare_line(mainClass.java:111)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CekShare_line -->
        <test-method status="PASS" signature="setDown()[pri:0, instance:rctiplus.mainClass@548e7350]" name="setDown" is-config="true" duration-ms="1417" started-at="2019-11-13T16:34:58Z" finished-at="2019-11-13T16:34:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setDown -->
      </class> <!-- rctiplus.mainClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
